src/org/zaproxy/zap/extension/importurls/resources/help_fil_PH/contents/importUrls.html

```diff
@@ -2,14 +2,14 @@
 <html>
 <head>
 <title>
-Import URLs
+Mag-import ng mga URL
 </title>
 </head>
 <body bgcolor="#ffffff">
-<h1>Import URLs</h1>
-This add-on adds an option to import a file of URLs  via the 'Tools' menu. The file must be plain text with one URL per line.
+<h1>Pag-import ng mga URL</h1>
```

Comment:
Granted I know nothing about this language but wouldn't the title and this heading be the same string?

Reply:
Yeah, this changes the meaning of the section. Something that should be checked/handled by proofreaders in Crowdin? Should we change Crowdin to export just approved translations?